apps/accounts/api/v1/views.py

```
@@ -0,0 +1,60 @@
+
+class UpdateUserView(generics.UpdateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UpdateUserSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class LoginView(knox_views.LoginView):
+    permission_classes = (AllowAny,)
+    serializer_class = LoginSerializer
+
+    def post(self, request, format=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `post`

(ANN201)